.storybook/Container.jsx

```diff
@@ -35,9 +34,9 @@ export default function Container({
           <CompatRouter>
             <Switch>
               <CompatRoute path={path}>
-                <React.StrictMode>
+                <StrictMode>
```

Comment:
Does `StrictMode` need to be imported above? As in: https://github.com/tektoncd/dashboard/pull/3399/files#diff-81b5d1056211d64bc68dcdd11a56379d3eaf54e3987fbb59e3e7a32f68da85f5R15

Reply:
I forgot to include a bunch of my changes in the initial commit, there's a second commit that does a bit of renaming and includes this.